python/elder/Cursoemvideo/ex006.py

```diff
@@ -0,0 +1,2 @@
+n = int(input('Digite um número:'))
+print(f'O número informado é {n} \no seu dobro é {n*2} \no seu triplo é {n*3} \nA raiz quadrada desse número {n} é {n**(1/2):.2f}')
```

Comment:
Quando vc tiver uma string longa assim, vc pode usar aspas triplas, mais ou menos assim:

```suggestion
texto = f"""
  O número informado é {n}
  O seu dobro é {n*2}
  O seu triplo é {n*3}
  A raiz quadrada desse número {n} é {n**(1/2):.2f}
"""
print(f'')
```